### PR TITLE
Hotfix/update-raster

### PR DESCRIPTION
**Pull request by krishnaglodha**
Updated coverages to work with local Tartu NTL data

### CODE REVIEW

File: workshop/exercises/pygeoapi.config.yml [hunk 1]
```diff
@@ -254,29 +254,29 @@ resources:
...
+    #     type: collection
+    #     title: Night Time Light Data 
+    #     description: Night Time Light Data averaged for 2023 in Tartu region.
+    #     keywords:
+    #         - Night Time Light
+    #     links:
+    #         -   type: text/html
+    #             rel: canonical
+    #             title: Nasa's Black Marble
+    #             href: https://blackmarble.gsfc.nasa.gov/
```
Comment (tomkralidis) [inline workshop/exercises/pygeoapi.config.yml:267]: Can we point to a more detailed link about the data source?

File: workshop/exercises/pygeoapi.config.yml [hunk 2]
```diff
@@ -254,29 +254,29 @@ resources:
...
+    #             hreflang: it
+    #     extents:
+    #         spatial:
+    #             bbox: [26.6264,58.32569 , 26.82632,58.433989]
```
Comment (tomkralidis) [inline workshop/exercises/pygeoapi.config.yml:271]: Remove spaces: `[26.6264,58.32569,26.82632,58.433989]`

**Review by tomkralidis — CHANGES_REQUESTED** [review]
@krishnaglodha changes requested.  Also, the exercise text (https://github.com/geopython/diving-into-pygeoapi/blob/main/workshop/content/docs/publishing/ogcapi-coverages.md) needs to be updated to reflect the new coverage data sample.

**Review by justb4 — CHANGES_REQUESTED** [review]
When review comments from @tomkralidis resolved GTG!

File: workshop/exercises/pygeoapi.config.yml (hunk 1 above)
Comment (krishnaglodha) [reply workshop/exercises/pygeoapi.config.yml:267]: there is no direct link, since we need to download it manually

**Comment by justb4** [comment]
Looks good now. For my undestanding: I run the exercise but apart from the data extent I don't see any raster data rendering from the GoeTIFF in the demo page: http://localhost:5000/collections/tartu-ntl . Or is that not intended (same I see now with Firenze data from 2022)?

**Comment by tomkralidis** [comment]
@justb4 correct, We are not rendering coverages. I will do a final review and merge later today.

**Pull request merged**